lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -17,12 +17,25 @@
 
 from .. import HYPERV
 from .context import NodeContext, get_node_context
+from .features import SecurityProfile
 from .hyperv_device_pool import HyperVDevicePool
 from .schema import HypervNodeSchema, HypervPlatformSchema
 from .serial_console import SerialConsole, SerialConsoleLogger
 from .source import Source
 
 
+def convert_to_hyperv_node_space(node_space: schema.NodeSpace) -> None:
+    """
+    Convert generic FeatureSettings to Hyper-V-specific types.
+    It converts generic FeatureSettings (like SecurityProfile) to platform-specific
+    types that have proper typing and platform-specific behavior.
+    """
+    if not node_space:
+        return
+
+    feature.reload_platform_features(node_space, HypervPlatform.supported_features())
```

Comment:
Which method raised the NotImplementedError? If it's part of a general implementation, we can move it to the base class.